### ISSUE TITLE
data structure varies depending upon number of rows in input

### ISSUE THREAD

**Issue by davedgd** [ddelange/mapply#7]
Thanks for putting together this handy package!

I noticed a small inconsistency in terms of how the data are returned when using `axis = 1` vs. pandas' apply in cases where the rows are < 200 vs. >= 200. Here's a minimal reproducible example of the issue:

```python
import pandas as pd
import mapply

mapply.init(n_workers = 2)

dat1 = pd.DataFrame(list(range(1, 200)))
dat2 = pd.DataFrame(list(range(1, 201)))

display([len(dat1), len(dat2)])

display([type(dat1.mapply(sum, axis = 1)), type(dat2.mapply(sum, axis = 1))])

display([type(dat1.apply(sum, axis = 1)), type(dat2.apply(sum, axis = 1))])
```

```python
[199, 200]
1/? [00:00<00:00, 19.46it/s]
2/? [00:00<00:00, 54.05it/s]
[pandas.core.frame.DataFrame, pandas.core.series.Series]
[pandas.core.series.Series, pandas.core.series.Series]
```

Note that with mapply one result is a DataFrame whereas the other is a Series, but for pandas apply returns a Series in both cases.

**Comment by ddelange** [comment]
Hi @davedgd,

The default `chunk_size=100`, combined with `n_workers=2` and the `dat1` frame with length 199, somehow prevents correct dtype casting.

Will try to have a look tonight.

Thanks for your bug report!

**Comment by davedgd** [comment]
Thank you @ddelange!

**Comment by ddelange** [comment]
Released [`0.1.6`](https://github.com/ddelange/mapply/releases/tag/0.1.6)

Thanks again :)
